include/aspect/simulator_access.h

```diff
@@ -256,6 +256,17 @@ namespace aspect
       convert_output_to_years () const;
 
       /**
+       * Return the number of the current pre refinement step.
+       * This can be useful for plugins that want to function differently in
+       * the initial adaptive refinements and later on.
+       * This will be not initialized before Simulator<dim>::run() is called.
+       * It equals 0 to parameters.initial_adaptive_refinement during the
```

Comment:
"It equals" is a bit hard to understand. How about "It iterates upward from 0 to ..."?